frontend/vre/annotation/annotation.model.js

```diff
@@ -1,71 +1,134 @@
-import {
-    getDateTimeLiteral,
-    JsonLdModel,
-    JsonLdNestedCollection,
-    jsonLdSync,
-    priorMethod,
-} from "../utils/jsonld.model";
-import {UserLd} from "../user/user.ld.model";
-import {glossary} from "../utils/glossary";
+import _ from 'lodash';
+import { parent } from '@uu-cdh/backbone-collection-transformers/src/inheritance.js';
 
+import {getDateTimeLiteral, JsonLdModel, JsonLdNestedCollection} from '../utils/jsonld.model';
+import {UserLd} from '../user/user.ld.model';
+import {glossary} from '../utils/glossary';
+
+// Helper function for removing multiple properties from an object in iteration.
+function stripAttribute(container) {
+    return function(name) {
+        delete container[name];
+    };
+}
+
+/**
+ * Backbone.Model representation of a Web Annotation to an EDPOP record.
+ *
+ * This model stores and synchronizes an annotation in compacted JSON-LD format.
+ * The Web Annotation data model is highly nested, which is inconvenient in a
+ * presentation and editing context. To remedy this, nested properties of
+ * interest are copied to the root `attributes` object during parsing. They are
+ * re-nested before being sent to the backend. Client code should read and edit
+ * these root-level attributes, rather than their standard nested paths (in parentheses):
+ *
+ * motivation            ([oa:motivatedBy][@id])
```

Comment:
Is there a reason why you didn't keep `oa:motivatedBy` here?

Reply:
By the way, very nice to have this documentation comment.

Reply:
Glad you like the documentation. Yes, the reason I don't reuse `oa:motivatedBy` is because that would be ambiguous (it is always present on the annotation itself, so if it is a string, is that because we flattened it or because the type changed from IRI to literal?). The flattened attributes in which I preserved the `oa:` prefixed name all come from deeper in the structure. I used `tagURL` instead of reusing `oa:hasBody` for the same reason.